route53_swatt_app.tf

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# Resource records that support SADR
```

Comment:
I'm assuming this comment should be updated to mention whatever "swatt" is?  If it's an acronym, please include what it stands for.  Even if not, please include a brief summary of swatt.

Reply:
SWATT = SoftWare ATTestation (just an abbreviate for the project).  Will eventually remove the SADR entry as that acronym has been deemed unacceptable by EA.

Reply:
@dav3r Fixed, thanks for catching that!

Reply:
See [18c9f13](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/88/commits/18c9f13ba2fe6c83b99b5dbcb26ddd8dcc323cab)